src/imap.rs

```diff
@@ -1607,13 +1607,6 @@ impl Session {
                 let encrypted_device_token_len = encrypted_device_token.len();
 
                 if encrypted_device_token_len <= 4096 {
```

Comment:
Since we store locally now, should we remove this check? Maybe move it to line 1644 where the token is pushed to the server